darts/timeseries.py

```diff
@@ -492,12 +513,12 @@ def from_series(pd_series: pd.Series,
         TimeSeries
             A TimeSeries constructed from the inputs.
         """
-        return TimeSeries(pd.DataFrame(pd_series), freq, fill_missing_dates)
+        return TimeSeries(pd.DataFrame(pd_series), freq, fill_missing_dates, column)
 
     @staticmethod
     def from_dataframe(df: pd.DataFrame,
-                       time_col: Optional[str],
-                       value_cols: Union[List[str], str],
+                       time_col: Optional[str] = None,
+                       value_cols: Optional[Union[List[str], str]] = None,
                        freq: Optional[str] = None,
                        fill_missing_dates: Optional[bool] = True) -> 'TimeSeries':
```

Comment:
As a bonus `from_dataframe` allow now to take only a single parameter (the dataframe) and infer everything (indices = time values) and columns become the multivariate values